src/js/components/Layer/stories/Round.js

```diff
@@ -61,6 +62,7 @@ export const RoundLayer = () => {
           position={position}
           onClickOutside={onClose}
           onEsc={onClose}
+          id="new layer"
```

Comment:
```suggestion
```
we can remove this since you've already demo'd the new behavior in a different story.